test/model/test_node.py

```diff
@@ -189,6 +189,13 @@ def test_from_state(client):
     assert node.last_seen == datetime(2023, 7, 18, 15, 42, 34, 701000, UTC)
 
 
+async def test_last_seen(lock_schlage_be469):
+    """Test last seen property."""
+    assert lock_schlage_be469.last_seen == datetime(
```

Comment:
Can we also test the statistics? Ie that there's this attribute in the node statistics even if the statistics data is missing this item.